modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/IgnitePersistentStoreDataStructuresTest.java

```diff
@@ -43,6 +46,9 @@
     /** */
     private static final TcpDiscoveryIpFinder ipFinder = new TcpDiscoveryVmIpFinder(true);
 
+    /** */
+    private static volatile boolean autoActivationEnabled = false;
```

Comment:
I don't think you really need volatile here. Nodes are started from single thread aren't they?

Reply:
@alamar , thank you for the review. `beforeTest` and `getConfiguration` methods can be called by different threads.

Reply:
If it's the case it should be a major eye-opener since I'm sure I've seen a few tests which may fail due to this.